.air.toml

```diff
@@ -0,0 +1,32 @@
+root = "."
+tmp_dir = "/tmp"
+
+[build]
+bin = "caddy run"
+cmd = "xcaddy build --with github.com/silinternational/auth-proxy=./"
```

Comment:
Note the `=./` at the end of the build command. This overrides with local files so you're not building from the `master` branch on Github.